PeopleAndAppleStockPrices/PeopleAndAppleStockPrices/StocksDetailVC.swift

```diff
@@ -0,0 +1,56 @@
+//
+//  StocksDetailVC.swift
+//  PeopleAndAppleStockPrices
+//
+//  Created by Sam Roman on 9/4/19.
+//  Copyright © 2019 Pursuit. All rights reserved.
+//
+
+import UIKit
+
+class StocksDetailVC: UIViewController {
+    
+    var previousStock: Stock!
+    var selectedStock: Stock!
+    
+    @IBOutlet weak var dateLabel: UILabel!
+    
+    @IBOutlet weak var stockImageView: UIImageView!
+    
+    @IBOutlet weak var openLabel: UILabel!
+    
+    @IBOutlet weak var closeLabel: UILabel!
+    
+    
```

Comment:
think your logic is backwards for this. the comparison shouldnt be the previous vs selected. It should be the open and close of the current stock. So if the open is greater than the close it should be a thumbs up.